Honeywell Lyric Integration (#14409)
home-assistant/home-assistant.github.io: source/_integrations/lyric.markdown

---
title: Honeywell Lyric
description: How to integrate the Honeywell Lyric integration into Home Assistant.
ha_category:
  - Climate
  - Sensor
ha_release: 2021.3
ha_iot_class: Cloud Polling
ha_config_flow: true
ha_codeowners:
  - '@timmo001'
ha_domain: lyric
---

The Honeywell Lyric component integrates the Lyric thermostat platform into Home Assistant.

## Setup

To set up this component, you will need to set up a developer account with Honeywell:

1. Go to the [developer site](https://developer.honeywellhome.com) and register with an account.
1. Next, create a [new app](https://developer.honeywellhome.com/user/me/apps/add) via the `My Apps` section.
   1. App Name: You can use any name here.

>    1. Callback URL: This needs to be the base URL of your Home Assistant instance + `/auth/external/callback`. For example: `http://192.168.1.123:8123/auth/external/callback`.

Reviewer: A small tweak, as the base URL doesn't matter anymore for OAuth 👍 

```suggestion
   1. Callback URL: This needs to be the URL of your Home Assistant instance + `/auth/external/callback`. For example: `http://192.168.1.123:8123/auth/external/callback`.
```

---
title: Honeywell Lyric
description: How to integrate the Honeywell Lyric integration into Home Assistant.
ha_category:
  - Climate
  - Sensor
ha_release: 0.116
ha_iot_class: Cloud Polling
ha_config_flow: true
ha_codeowners:
  - '@timmo001'
ha_domain: lyric
---

The Honeywell Lyric component integrates the Lyric thermostat platform into Home Assistant.

## Setup

To set up this component, you will need to set up a developer account with Honeywell:

1. Go to the [developer site](https://developer.honeywellhome.com) and register with an account.
1. Next, create a [new app](https://developer.honeywellhome.com/user/me/apps/add) via the `My Apps` section.
   1. App Name: You can use any name here.
   1. Callback URL: This needs to be the base URL of your Home Assistant instance + `/auth/external/callback`. For example: `http://192.168.1.123:8123/auth/external/callback`.

Once your app is created, copy the `Consumer Key` and `Consumer Secret`. These will be your `client_id` and `client_secret` to add to your configuration:

```yaml
# Example configuration.yaml entry
lyric:

>   client_id: myconsumerkey
>   client_secret: myconsumersecret

Reviewer: ```suggestion
  client_id: MY_CONSUMER_KEY
  client_secret: MY_CONSUMER_SECRET
```

```

You can then add the integration in the frontend.

## Configuration

Menu: **Configuration** -> **Integrations**.

Click on the `+` sign to add an integration and click on **Honeywell Lyric**.

> Login with your Honeywell Lyric account and agree to the terms. The Honeywell Lyric integration will then be available.

Reviewer: ```suggestion
Log in with your Honeywell Lyric account and agree to the terms. The Honeywell Lyric integration will then be available.
```
